webpack.config.develop.js

```diff
@@ -4,6 +4,22 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin')
 const { PORT } = require('./config')
 
 module.exports = {
+  module: {
```

Comment:
What's the reason for changing the way this was defined? Using the `IS_DEV` flag and having the config in one place seemed neater?

Reply:
I was trying to move to just using the files (`config.production` or `config.development`) rather than using the `IS_DEV` variable and the files for specifying different configs for `production` or `development`. But agreed on reflection this seems a little verbose for the sake of it - have reverted to what was there 👍